### PR TITLE
feat: add `constants/float32/ln-ten`

### LINKED ISSUE #3343: [RFC]: add `constants/float32/ln-ten`
### Description

This RFC proposes
- This RFC proposes adding the package @stdlib/constants/float32/ln-ten returning Natural Logarithm of single-precision floating-point number 10.

Image

### Related Issues

Related issues None.

### Questions

No.

### Other

No.

### Checklist

- [x] I have read and understood the [Code of Conduct](https://github.com/stdlib-js/stdlib/blob/develop/CODE_OF_CONDUCT.md).
- [x] Searched for existing issues and pull requests.
- [x] The issue name begins with `RFC:`.

### PR DESCRIPTION
**Pull request by vivekmaurya001**
Resolves : #3343

## Description

> What is the purpose of this pull request?

This pull request:

-   This RFC proposes adding the package @stdlib/constants/float32/ln-10 returning Natural Logarithm of single-precision floating-point number 10.

## Related Issues

> Does this pull request have any related issues?

This pull request:

-   resolves #{{TODO: add issue number}} 

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

 - [x] Read, understood, and followed the [contributing guidelines](https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md)

* * *

@stdlib-js/reviewers 

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

### CODE REVIEW

File: lib/node_modules/@stdlib/constants/float32/ln-ten/README.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# FLOAT32_LN10
+
+> [Natural logarithm][@stdlib/math/base/special/ln] of single-precision floating-point number `10`.
```
Comment (kgryte) [inline lib/node_modules/@stdlib/constants/float32/ln-ten/README.md]:
```suggestion
> [Natural logarithm][@stdlib/math/base/special/ln] of `10` as a single-precision floating-point number.
```

This change applies here and elsewhere throughout this PR.

File: lib/node_modules/@stdlib/constants/float32/ln-ten/lib/index.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+*
+* ```tex
+* \ln 10
+* ```
+*
+* @constant
+* @type {number}
+* @default 2.3025851249694824
+*/
+var FLOAT32_LN10 = float64ToFloat32(2.30258509299404568401799145468436420760110148862877297603332790096757260967735248023599720508959829834196778404e+00); // eslint-disable-line max-len
```
Comment (kgryte) [inline lib/node_modules/@stdlib/constants/float32/ln-ten/lib/index.js]:
```suggestion
var FLOAT32_LN10 = float64ToFloat32( 2.30258509299404568401799145468436420760110148862877297603332790096757260967735248023599720508959829834196778404e+00 ); // eslint-disable-line max-len
```

File: lib/node_modules/@stdlib/constants/float32/ln-ten/test/test.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+// TESTS //
+
+tape( 'main export is a number', function test( t ) {
+	t.ok( true, __filename );
+	t.strictEqual( typeof FLOAT32_LN10, 'number', 'main export is a number' );
+	t.end();
+});
+
+tape( 'export is a single-precision floating-point number equal to ` 2.3025851249694824`', function test( t ) {
```
Comment (kgryte) [inline lib/node_modules/@stdlib/constants/float32/ln-ten/test/test.js]:
```suggestion
tape( 'export is a single-precision floating-point number equal to `2.3025851249694824`', function test( t ) {
```

File: lib/node_modules/@stdlib/constants/float32/ln-ten/test/test.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+	t.equal( FLOAT32_LN10, lnf( 10.0 ), 'equals  2.3025851249694824' );
```
Comment (kgryte) [inline lib/node_modules/@stdlib/constants/float32/ln-ten/test/test.js]:
```suggestion
	t.equal( FLOAT32_LN10, 2.3025851249694824, 'returns expected value' );
```

File: lib/node_modules/@stdlib/constants/float32/ln-ten/docs/repl.txt [hunk 5]
```diff
@@ -0,0 +1,12 @@
+
+{{alias}}
+    Natural logarithm of single-precision floating-point number `10`.
```
Comment (kgryte) [inline lib/node_modules/@stdlib/constants/float32/ln-ten/docs/repl.txt]: The description changes made in the README should be made here and elsewhere throughout this PR.

Comment (vivekmaurya001) [reply lib/node_modules/@stdlib/constants/float32/ln-ten/docs/repl.txt]: done

Comment (vivekmaurya001) [reply lib/node_modules/@stdlib/constants/float32/ln-ten/docs/repl.txt]: done

**Review by kgryte — APPROVED** [review]

**Pull request merged**